numerapi/numerapi.py

```diff
@@ -112,6 +114,73 @@ def download_current_dataset(self, dest_path=".", dest_filename=None,
 
         return dataset_path
 
+    def get_latest_data_path(self, data_type: str, ext: str = "csv") -> str:
```

Comment:
maybe rename from `path` to `url`, to make it more explicit that this needs to be downloaded